### DISCUSSION TITLE
Environment variables inside r code

### DISCUSSION THREAD

**Discussion by Juuso1** [quarto-dev/quarto-cli#5593] [Q&A] [answered]
Hi,

How can use environment variables inside r code?

For example database server below
# database server
DATABASE_SERVER=https://db.example.com

BR,
Juuso

**Comment by mcanouil** [comment, marked as answer]
What does this have to do with Quarto?
You'll have better chances to ask this on R dedicated platform.

This being said: `Sys.getenv()` (note that if you define the variable in R, you can only access it within the same session).

**Reply by Juuso1** [reply]
Hi,

Thanks, it worked. 
stat_server = Sys.getenv("STAT_SERVER")
